### PR TITLE
Add color handling methods

### PR DESCRIPTION
**Pull request by rtgill82**
Add ability to change newt colors globally or individually for implemented widgets that support it (Label and Textbox).

### CODE REVIEW

File: examples/test_method/Screen_set_colors.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+#require 'rubygems'
```
Comment (lzap) [inline examples/test_method/Screen_set_colors.rb]: Hello, thanks for your contributiion. This is leftovere.

**Comment by lzap** [comment]
Looks good, can you squash into one commit please?

**Comment by rtgill82** [comment]
Done.

**Comment by lzap** [comment]
Thank you, I released 0.9.7.

**Pull request merged**